### DISCUSSION TITLE
🚨 Docker container user heeft niet genoeg rechten

### DISCUSSION THREAD

**Discussion by attaxia** [dsmrreader/dsmr-reader#1553] [Questions & answers / How-to's] [answered]
### Description

Drie dagen geleden de docker container geupgrade, verder geen wijzigingen. Sindsdien geen statistieken meer.
Logs staan vol met:

`RuntimeError: ('Failed to connect: {}', SerialException(13, "could not open port /dev/serial/by-id/usb-FTDI_FT232R_USB_UART_AI2BAI4B-if00-port0: [Errno 13] Permission denied: '/dev/serial/by-id/usb-FTDI_FT232R_USB_UART_AI2BAI4B-if00-port0'"))
`

Ik heb het probleem op kunnen lossen door het volgende commando (in de container) uit te voeren:
sudo adduser app dialout

### DSMR-reader version

4.19

### DSMR-reader platform

Docker

**Comment by dennissiemensma** [comment]
Je kunt het beste hiervoor een issue aanmaken in het docker project van xirixiz. Vh5eck ook even of er wellicht anderen zijn die eerder hetzelfde gemeld hebben.

**Comment by dennissiemensma** [comment]
Zie mogelijk https://github.com/xirixiz/dsmr-reader-docker/issues/251

**Comment by xirixiz** [comment, marked as answer]
Pleasy try the latest development release of the Docker image (ready in about 30 minutes from now).

**Comment by poerlemans** [comment]
The development release of the docker image looks solving the problem.
